docs/src/dev-playground.md

```
@@ -0,0 +1,138 @@
+=== "Docker-Desktop"
+
+    Enable it in Docker-Desktop
+
+
+
+
+## 2. Start a Local Docker Registry
+
+Run a local Docker registry container, to push our images locally and access them in our cluster.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)